src/dom/query.ts

```diff
@@ -86,12 +88,26 @@ function hitTest(node: Element, clientX: number, clientY: number): boolean {
  * ```
  */
 export
-function scrollIfNeeded(area: HTMLElement, elem: HTMLElement, threshold = 0): void {
+function scrollIfNeeded(area: HTMLElement, elem: HTMLElement): void {
```

Comment:
`IfNeeded` is a bad name anyway -- what need are we talking about? This will scroll even if the element is partly visible, so it's not "if needed to see some part of the element".